Stylesheets/pastelSolarized.nb

```diff
@@ -0,0 +1,287 @@
+ "SyntaxErrorStyle"->{FontColor -> RGBColor[0.862745, 0.196078, 0.184314]},
+ "UndefinedSymbolStyle"->{FontColor -> RGBColor[0.14902, 0.545098, 0.823529]}},
+ FontFamily->"Inconsolata-dz",
+ FontSize->12,
+ FontWeight->"Plain",
+ FontSlant->"Plain"],
+
+Cell[StyleData["Input"],
+ CellFrame->1,
+ CellMargins->{{65, 0}, {0, 5}},
+ CellGroupingRules->{GroupTogetherGrouping, 10000.},
+ CellFrameColor->RGBColor[0.933333, 0.909804, 0.835294],
+ AutoIndent->False,
+ CellChangeTimes->{3.545825155715643*^9},
+ AutoSpacing->True,
+ FontWeight->"Plain",
+ FontColor->RGBColor[0.027451, 0.211765, 0.258824],
+ Background->RGBColor[0.992157, 0.964706, 0.890196]],
+
+Cell[CellGroupData[{
+
+Cell[StyleData["Output"],
+ CellFrame->1,
+ CellMargins->{{65, 0}, {5, -1}},
+ CellGroupingRules->{GroupTogetherGrouping, 10000.},
+ CellFrameColor->RGBColor[0.933333, 0.909804, 0.835294],
+ CellChangeTimes->{3.545825155716502*^9},
+ Background->GrayLevel[1]],
+
+Cell[StyleData["Input", "Printout", StyleDefinitions -> "None"],
+ CellMargins->{{39, 10}, {0, 7}},
+ CellFrameLabelMargins->0,
+ LinebreakAdjustments->{0.85, 2, 10, 1, 1},
+ Background->RGBColor[0.992157, 0.964706, 0.890196]],
+
+Cell[StyleData["Output", "Printout", StyleDefinitions -> None],
+ CellMargins->{{39, 10}, {7, 0}},
+ CellFrameLabelMargins->0,
+ Background->GrayLevel[1]]
+}, Open  ]],
+
+Cell[StyleData["Code"],
+ PageWidth->WindowWidth,
+ CellFrame->1,
+ CellMargins->{{65, 0}, {0, 5}},
+ CellFrameColor->RGBColor[0.622222, 0.606536, 0.556863],
+ FontWeight->"Plain",
+ Background->RGBColor[0.992157, 0.964706, 0.890196]],
+
+Cell[StyleData["PrintUsage"],
+ Editable->False,
+ PageWidth->WindowWidth,
+ CellFrame->{{0, 0}, {0, 2}},
+ CellFrameColor->RGBColor[0.14902, 0.545098, 0.823529],
+ ContextMenu->FEPrivate`FrontEndResource["ContextMenus", "Output"],
+ Background->RGBColor[0.887937, 0.940095, 0.976761]]
+}, Open  ]],
+
+Cell[CellGroupData[{
+
+Cell["Titles, sections and text", "Section",
+ CellChangeTimes->{{3.5472314281356163`*^9, 3.547231435673066*^9}, 
+   3.547231487937338*^9, {3.547232227015649*^9, 3.54723223069147*^9}}],
+
+Cell[StyleData["Title"],
+ FontFamily->"Optima",
+ FontColor->RGBColor[0.14902, 0.545098, 0.823529]],
+
+Cell[StyleData["Subtitle"],
+ FontFamily->"Optima"],
+
+Cell[StyleData["Subsubtitle"],
+ CellGroupingRules->{GroupTogetherGrouping, 10000.},
+ CellChangeTimes->{3.5531667905651913`*^9},
+ FontFamily->"Optima"],
+
+Cell[StyleData["Section"],
+ CellGroupingRules->{GroupTogetherGrouping, 10000.},
+ CellChangeTimes->{3.5531667905640497`*^9},
+ FontFamily->"Optima"],
+
+Cell[StyleData["Subsection"],
+ CellGroupingRules->{GroupTogetherGrouping, 10000.},
+ CellChangeTimes->{3.5531667905648108`*^9},
+ FontFamily->"Optima"],
+
+Cell[StyleData["Subsubsection"],
+ FontFamily->"Optima"],
+
+Cell[StyleData["Text"],
+ TextJustification->1,
+ FontFamily->"Optima",
+ FontSize->12]
+}, Open  ]]
+},
+WindowSize->{740, 756},
+WindowMargins->{{Automatic, 64}, {Automatic, 0}},
```

Comment:
I had bad experiences with these settings. While here it looks OK, it was possible (I use 2 screens) at least in earlier versions that a notebook was off the screen when opening it.